cmd/all-in-one/all_in_one_test.go

```diff
@@ -27,16 +27,19 @@ import (
 // An optional SKIP_SAMPLING=true environment variable can be used to skip sampling checks (for jaeger-v2).
 
 const (
-	host      = "0.0.0.0"
```

Comment:
please delete line 27, it's no longer relevant

Reply:
you can pass a similar env variable if you need to customize the behavior for v2 health check.

Reply:
For example, see how it was previously done (removed in #5802)

Reply:
Ok, thanks!